src/components/AppBar.jsx

```diff
@@ -1,15 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
-import { fade, makeStyles, useTheme } from '@material-ui/core/styles';
+import {
+  fade,
+  makeStyles,
+  withStyles,
+  useTheme
+} from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
```

Comment:
Let's remove the `PropTypes` import for now, since we don't have any props yet.

Reply:
Oops, my bad. I added it, took it out, but forgot to remove the import.